src/main/java/com/yahoo/bullet/record/avro/UntypedAvroBulletRecord.java

```diff
@@ -65,7 +65,7 @@ public UntypedAvroBulletRecord remove(String field) {
     @Override
     public UntypedAvroBulletRecord copy() {
         UntypedAvroBulletRecord copy = new UntypedAvroBulletRecord();
-        copy.data = new LazyBulletAvro(this.data);
+        copy.data = this.data.copy();
```

Comment:
This breaks the symmetry with TypedAvroBulletRecord, but we could do `return new UntypedAvroBulletRecord(data.copy())` which saves us from instantiating the empty LazyBulletAvro() in the noargs.

Reply:
Oh right, there's an allargsconstructor in TypedAvroBulletRecord so you could do something similar there? 🤔